sotu.md

```diff
@@ -324,6 +324,7 @@ features such as:
   building
 * [`process`](https://hackage.haskell.org/package/process) - low-level library for sub-process management
 * [`ansi-terminal`](https://hackage.haskell.org/package/ansi-terminal) - de facto standard cross-platform terminal library (works on Windows as well)
+* [`path`](https://github.com/commercialhaskell/path) - type safe handling of file paths
```

Comment:
I think it's better to keep consistency and use hackage package link (https://hackage.haskell.org/package/path) instead of github repo link..